contracts/modules/CompoundManager.sol

```diff
@@ -87,184 +75,6 @@ contract CompoundManager is BaseModule, RelayerModule, OnlyOwnerModule {
-    {
-        address cToken = compoundRegistry.getCToken(_collateral);
-        redeemUnderlying(_wallet, cToken, _collateralAmount);
-        exitMarketIfNeeded(_wallet, cToken, address(comptroller));
-        emit CollateralRemoved(address(_wallet), _loanId, _collateral, _collateralAmount);
-    }
-
-    /**
-     * @dev Increases the debt by borrowing more token from a loan identified by its ID.
-     * @param _wallet The target wallet.
-     * @param _loanId bytes32(0) as Compound does not allow the creation of multiple loans.
-     * @param _debtToken The token borrowed.
-     * @param _debtAmount The amount of token to borrow.
-     */
-    function addDebt(
-        BaseWallet _wallet,
-        bytes32 _loanId,
-        address _debtToken,
-        uint256 _debtAmount
-    )
-        external
-        onlyWalletOwner(_wallet)
-        onlyWhenUnlocked(_wallet)
-    {
-        address dToken = compoundRegistry.getCToken(_debtToken);
-        enterMarketIfNeeded(_wallet, dToken, address(comptroller));
-        borrow(_wallet, dToken, _debtAmount);
-        emit DebtAdded(address(_wallet), _loanId, _debtToken, _debtAmount);
-    }
-
-    /**
-     * @dev Decreases the debt by repaying some token from a loan identified by its ID.
-     * @param _wallet The target wallet.
-     * @param _loanId bytes32(0) as Compound does not allow the creation of multiple loans.
-     * @param _debtToken The token to repay.
-     * @param _debtAmount The amount of token to repay.
-     */
-    function removeDebt(
-        BaseWallet _wallet,
-        bytes32 _loanId,
-        address _debtToken,
-        uint256 _debtAmount
-    )
-        external
-        onlyWalletOwner(_wallet)
-        onlyWhenUnlocked(_wallet)
-    {
-        address dToken = compoundRegistry.getCToken(_debtToken);
-        repayBorrow(_wallet, dToken, _debtAmount);
-        exitMarketIfNeeded(_wallet, dToken, address(comptroller));
-        emit DebtRemoved(address(_wallet), _loanId, _debtToken, _debtAmount);
-    }
-
-    /**
-     * @dev Gets information about the loan status on Compound.
-     * @param _wallet The target wallet.
-     * @return a status [0: no loan, 1: loan is safe, 2: loan is unsafe and can be liquidated]
-     * and a value (in ETH) representing the value that could still be borrowed when status = 1; or the value of the collateral
-     * that should be added to avoid liquidation when status = 2.
-     */
-    function getLoan(
-        BaseWallet _wallet,
-        bytes32 /* _loanId */
-    )
-        external
-        view
-        returns (uint8 _status, uint256 _ethValue)
-    {
-        (uint error, uint liquidity, uint shortfall) = comptroller.getAccountLiquidity(address(_wallet));
-        require(error == 0, "Compound: failed to get account liquidity");
-        if (liquidity > 0) {
-            return (1, liquidity);
-        }
-        if (shortfall > 0) {
-            return (2, shortfall);
-        }
-        return (0,0);
-    }
-
```

Comment:
Why are you removing all the Loan logic from CompoundManager? While it is not implemented in the clients yet it should still be on the roadmap. Since this code has been audited and we are still planning to use it at some point I don't see why we should remove it from the contract now.